application/src/main/webapp/vue-app/components/ExoChatMessageDetail.vue

```diff
@@ -403,6 +422,15 @@ export default {
         document.dispatchEvent(new CustomEvent(`exo-chat-message-action-${messageAction.key}-requested`, {detail: this.message}));
       }
     },
+    isURL(url) {
+      const pattern = new RegExp('^(https?:\\/\\/)?'+ // protocol
```

Comment:
What if we have a message with a link inside, we need to consider following cases too :  
 - URL preceeded and followed by text
 - Many URLs inside a single message

Reply:
I will consider this case